packages/web/public/static/locales/de/settings.json

```diff
@@ -155,7 +155,7 @@
       "advancedFiltering": "Filtere Beiträge mit erweiterten Filtern",
       "badges": "Erhalte für Meilensteine Abzeichen",
       "stats": "Verfolge deine Aktivitäten und Statistiken",
-      "unsplashIntegration": "Wähle Bilder von Unsplash (kommt bald)",
+      "unsplashIntegration": "Free images with Unsplash",
```

Comment:
If you just delete the translation here, it will fall back to the English copy (in prod, in dev it falls back to the message key for debugging)